us-geological-survey.csl

```diff
@@ -121,8 +150,8 @@
               <et-al term="and others"/>
               <label form="short" prefix=" "/>
             </names>
-            <text variable="container-title" prefix=" " suffix=","/>
-            <text variable="collection-title" prefix=" " suffix=","/>
+            <text variable="event" prefix=" " suffix=" --"/>
```

Comment:
the style really wants two ugly hyphens there? not a pretty little en dash or so?